include/odp/api/spec/queue.h

```diff
@@ -116,6 +116,10 @@ int odp_queue_context_set(odp_queue_t queue, void *context, uint32_t len);
 /**
  * Get queue context
  *
+ * Returns previously stored queue context pointer from the application.
```

Comment:
`from the application` part seems a bit out of place here.

Reply:
Fixed.